apps/protailwind/src/pages/workshops/[module]/index.tsx

```diff
@@ -1,33 +1,46 @@
 import React from 'react'
 import {SanityDocument} from '@sanity/client'
-import {GetStaticPaths, GetStaticProps} from 'next'
+import {GetStaticPaths, GetStaticProps, GetServerSideProps} from 'next'
 import {getAllWorkshops, getWorkshop} from '../../../lib/workshops'
 import WorkshopTemplate from '../../../templates/workshop-template'
+import {getActiveProducts} from 'path-to-purchase-react/products.server'
+import {propsForCommerce} from '@skillrecordings/commerce-server'
+import {getToken} from 'next-auth/jwt'
+import {CommerceProps} from '@skillrecordings/commerce-server/dist/@types'
 
 export const USER_ID_QUERY_PARAM_KEY = 'learner'
 
-export const getStaticProps: GetStaticProps = async ({params}) => {
+export const getServerSideProps: GetServerSideProps = async ({
+  res,
+  req,
+  query,
+  params,
+}) => {
+  res.setHeader('Cache-Control', 's-maxage=1, stale-while-revalidate')
```

Comment:
I don't think you can have this AND load `propsForCommerce` which is custom for the user/products etc
```suggestion
```